locales/en-US/server.ftl

```diff
@@ -125,7 +128,16 @@ shotPageEditButton =
     .title = Edit this image
 shotPagefavoriteButton =
     .title = Favorite this shot
+shotPageBackToHomeButton =
+    .title = Homepage
+shotPageAllShotsButton =
+    .title = All Shots
+shotPageAllShots = All Shots
 shotPageDownload = Download
+# Note: Draw, Favorite and Delete text below is used on shot page header buttons as a verb (action)
```

Comment:
Please use one comment per string (not really needed for 'delete', since there's no space for confusion). Localizers don't see the file, see one string at a time, and the associated comment.

The alternative is to use section comments, but I'd avoid that in this case.

Reply:
@flodolo Good to know, Done. Thanks!